superset/utils/core.py

```diff
@@ -811,7 +811,13 @@ def send_mime_email(
     smtp.quit()
 
 
-def get_email_address_list(address_string: str) -> list[str]:
+def get_recipients_list(address_string: str | None) -> list[str]:
```

Comment:
Since this isn't really fetching anything, can we change it to sound more like a utility, like "recipient_string_to_list"?

Reply:
Sure! I'll do that in another commit